includes/init.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
For sake of uniformity `yml` extension used for throughout this role, will be good to rename the file to `init.yml`